### ISSUE TITLE
Optional remove sourcemap

### ISSUE THREAD

**Issue by sugarshin** [bugsnag/webpack-bugsnag-plugins#2]
Hi,

I need `removeSourcemap` option, like https://github.com/bakunyo/bugsnag-sourcemap-webpack-plugin#options

Thank you.

**Comment by bengourley** [comment]
Thanks for the request @sugarshin, I've added this to our internal backlog.

**Comment by sugarshin** [comment]
Look forward to that.

**Comment by bohdanbirdie** [comment]
+1 here, would be great to have this option!

**Comment by thrstn** [comment]
+1 I really need this option.

**Comment by SayakMukhopadhyay** [comment]
This feature might be a small one but a very important one nonetheless. Sourcemaps *needs* to be generated for Bugsnag to work but they should never be served. Although one can workaround this, having an option to delete the maps would be great.

**Comment by asgeo1** [comment]
I'm using Netlify, so it pulls from my git repo directly, and builds and deploys. I don't think they have an option to filter out *.js.map files from `dist`, or to restrict access to them. I think deleting them after the upload to bugsnag is my best option.

So I can code that myself, but surely there's lots of use-cases like that.

I know what you're saying about it not being this plugins responsibility - but the only reason I turned on source-maps was to upload them to bugsnag, and something has to clean up these files afterwards... It would be a pretty convenient option to include. Hope you reconsider in the future!

**Comment by snmaynard** [comment]
With netlify you have a build command so you can do something like `npm run build && rm dist/*.map`?

**Comment by asgeo1** [comment]
Yes, I just tried that and it works 👍   In lieu of a more explicit option, something as simple as that will do the job. Cheers
